engines/director/sprite.cpp

```diff
@@ -472,10 +472,12 @@ void Sprite::setCast(CastMemberID memberID) {
 				}
 			}
 			break;
+		case kCastFilmLoop:
 		case kCastShape:
 		case kCastText: 	// fall-through
 			break;
 		default:
+			warning("Sprite::setCast(): Setting bbox of castId %s , type: %s to 0", memberID.asString().c_str(), castType2str(_cast->_type));
```

Comment:
Should this still be as a `warning()`, or is it better to hide it behind a `debug(3, kDebugImages. ...`?